pkg/kubelet/dockertools/manager.go

```diff
@@ -48,12 +48,12 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/network/hairpin"
 	proberesults "k8s.io/kubernetes/pkg/kubelet/prober/results"
 	"k8s.io/kubernetes/pkg/kubelet/qos"
-	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
```

Comment:
@sjpotter @dchen1107 If so, we should also change `kubecontainer` and `kubepod` in the future. :)